frontend/src/views/Transfers/__tests__/buttonConfigs.test.jsx

```diff
@@ -197,12 +210,216 @@ describe('buttonClusterConfigFn', () => {
+        }
+      }
+      const transferData = {
+        currentStatus: {
+          status: TRANSFER_STATUSES.RESCINDED
+        },
+        agreementDate: new Date(),
+        transferHistory: []
+      }
+
+      const config = buttonClusterConfigFn({
+        toOrgData,
+        hasRoles,
+        hasAnyRole,
+        currentUser,
+        methods,
+        t,
+        setModalData,
+        createUpdateTransfer,
+        transferData,
+        isGovernmentUser,
+        recommendation,
+        signingAuthorityDeclaration
+      })
+
+      const buttons = config[TRANSFER_STATUSES.RESCINDED]
+
+      expect(buttons).toHaveLength(0)
+    })
+  })
+
+  describe('When status is Declined', () => {
+    it('should return an empty array', () => {
+      const hasRoles = vi.fn()
+      const hasAnyRole = vi.fn()
+      const currentUser = {
+        isGovernmentUser: false,
+        organization: {
+          organizationId: 1
+        }
+      }
+      const transferData = {
+        currentStatus: {
+          status: TRANSFER_STATUSES.DECLINED
+        },
+        agreementDate: new Date(),
+        transferHistory: []
+      }
+
+      const config = buttonClusterConfigFn({
+        toOrgData,
+        hasRoles,
+        hasAnyRole,
+        currentUser,
+        methods,
+        t,
+        setModalData,
+        createUpdateTransfer,
+        transferData,
+        isGovernmentUser,
+        recommendation,
+        signingAuthorityDeclaration
+      })
+
+      const buttons = config[TRANSFER_STATUSES.DECLINED]
+
+      expect(buttons).toHaveLength(0)
+    })
+  })
+
+  describe('When status is Refused', () => {
+    it('should return an empty array', () => {
+      const hasRoles = vi.fn()
+      const hasAnyRole = vi.fn()
+      const currentUser = {
+        isGovernmentUser: false,
+        organization: {
+          organizationId: 1
+        }
+      }
+      const transferData = {
+        currentStatus: {
+          status: TRANSFER_STATUSES.REFUSED
+        },
+        agreementDate: new Date(),
+        transferHistory: []
+      }
+
+      const config = buttonClusterConfigFn({
+        toOrgData,
+        hasRoles,
+        hasAnyRole,
+        currentUser,
+        methods,
+        t,
+        setModalData,
+        createUpdateTransfer,
+        transferData,
+        isGovernmentUser,
+        recommendation,
+        signingAuthorityDeclaration
+      })
+
+      const buttons = config[TRANSFER_STATUSES.REFUSED]
+
+      expect(buttons).toHaveLength(0)
+    })
   })
 
   describe('When status is Sent', () => {
+    const transferData = {
+      currentStatus: {
+        status: TRANSFER_STATUSES.SENT
+      },
+      agreementDate: new Date(),
+      transferHistory: [],
+      toOrganization: {
+        organizationId: 2
+      },
+      fromOrganization: {
+        organizationId: 1
+      }
+    }
+
     it('should return declineTransfer and signAndSubmit buttons for toOrganization user with roles', () => {
```

Comment:
Similar to above, looks like this is no longer testing for the declineTransfer button